modules/installation-configuring-shiftstack-single-ipv6.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// TBD
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-shiftstack-single-ipv6_{context}"]
+= Configuring a cluster with single-stack IPv6 networking
+
+You can create a single-stack IPv6 cluster on {rh-openstack-first}.
+
+.Prerequisites
+
+* Your {rh-openstack} deployment has an existing IPv6 subnet.
+* DNS is configured for the existing IPv6 subnet.
+* The IPv6 subnet is added to a {rh-openstack} router, and the router is configured to send router advertisements (RAs). 
+* The {rh-openstack} network accepts a minimum MTU size of 1380 bytes.
```

Comment:
@MaysaMacedo We are testing with MTU 1422. Should we put that value here? Where is this 1380 value coming from?

Reply:
Leaving this. Unresolve if there are changes to make.

Reply:
I think we still needs Ramon input here, if I remember correctly he had to use a bigger MTU

Reply:
We had to use a bigger MTU in the underlying infrastructure: We needed to increase the MTU in the geneve interfaces while configuring the openstack computes in RHOSO. So I don't think this is really related to this limitation.

IMO, I would put 1442 here as that is the value we have been testing and we will have on the CI.

Reply:
```suggestion
* The {rh-openstack} network accepts a minimum MTU size of 1442 bytes.
```